Source/Core/VideoBackends/Software/SWOGLWindow.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2015 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include "Common/CommonTypes.h"
+
+class SWOGLWindow
+{
+public:
+	SWOGLWindow(void *window_handle);
```

Comment:
Pointer against type. Also, having this public instead of an initialization function when using a singleton pattern isn't particularly advisable.